aidants_connect_web/templates/aidants_connect_web/managment/notify_no_totp_workers.html

```diff
@@ -0,0 +1,34 @@
+{% extends "layouts/email_base.html" %}
+{% block email_body %}
+  <table border="0" cellpadding="0" cellspacing="0" class="body">
+    <tr>
+      <td>&nbsp;</td>
+      <td class="container">
+        <div class="content">
+          <table class="main">
+            <tr>
+              <td class="wrapper">
+                <table border="0" cellpadding="6" cellspacing="0">
+                  <tr>
+                    <td>
+                      <p>Bonjour,</p>
+                      <p>L’équipe Aidants Connect vous informe qu'aucune carte TOTP n'est associé aux aidants ou aidantes suivantes&nbsp;:</p>
```

Comment:
Le terme "associer" semblait poser problème à certains utilisateurs, on est partis sur le terme "lier" ailleurs dans l'interface : 

```suggestion
                      <p>L’équipe Aidants Connect vous informe qu'aucune carte TOTP n'est liée aux aidants ou aidantes suivantes&nbsp;:</p>
```

Reply:
Ah oui et on ne parle plus de carte TOTP parce que ça fait peur, mais de "Carte Aidants Connect" : 
```suggestion
                      <p>L’équipe Aidants Connect vous informe qu'aucune carte Aidants Connect n'est liée aux aidants ou aidantes suivantes&nbsp;:</p>
```